subcommands/targets/offline-update.go

```diff
@@ -562,6 +565,12 @@ func doShowBundle(cmd *cobra.Command, args []string) {
 	}
 
 	rootMeta, err := getLatestRoot(tufMetaPath)
+	if errors.Is(err, os.ErrNotExist) && bundleMeta.ouBundleMeta.Type == "ci" {
+		// If no any N.root.json is found in the bundle and this is the "ci" bundle,
+		// then this is the valid case - a user has not taken their TUF targets key offline.
+		// Therefore, instead of failing the command fetches the root meta from the backend.
+		rootMeta, err = api.TufRootGet(viper.GetString("factory"))
+	}
```

Comment:
Do you mean that TUF root versions 1 and 2 are never included into the update bundle?
This seems strange, but I might be out of context.

Reply:
Yes. From the security standpoint of view this the right things to do, otherwise anyone can generate their own chain of TUF keys and put into a bundle. Then, a non-registered device will trust it.
However, we protect our devices from it by pre-provisioning them with the first N root versions...